admin/app.py

```diff
@@ -4,42 +4,51 @@
 from bson import ObjectId
 from dotenv import load_dotenv
 from flask import flash, Flask, redirect, render_template, request, url_for
-from forms import AddUserForm, UpdateUserForm
+from flask_login import LoginManager, logout_user, login_user, login_required
 from mongoengine import connect
+from werkzeug.urls import url_parse
 from werkzeug.security import generate_password_hash
 
-from models import User
-
 load_dotenv()
 
+
+app = Flask(__name__)
+app.config['SECRET_KEY'] = os.getenv('SECRET_KEY')
 connect(
     db=os.getenv('DB_NAME'),
     host=os.getenv('MONGO_URL'),
     port=int(os.getenv('PORT'))
 )
+login = LoginManager(app)
+login.login_view = 'admin_login'
+login.init_app(app)
 
-app = Flask(__name__)
-app.config['SECRET_KEY'] = os.getenv('SECRET_KEY')
+from forms import LoginForm, AddUserForm, UpdateUserForm
```

Comment:
@AndriiSkavreniuk Move the imports up to the rest of imports

Reply:
i cant do this
ImportError: cannot import name 'login' from partially initialized module 'app' (most likely due to a circular import) (/home/andrii/PycharmProjects/Ch-117-Python-Project-ITA/admin/app.py)